cookbook/flytetester.json

```diff
@@ -0,0 +1,65 @@
+ [{
+    "name": "Core",
```

Comment:
we should add priority here too... so your workflow knows which ones to run when p0/p1/p2 is selected/desired